### PR TITLE
feature/webpack was build and the sidebar was build

### PR DESCRIPTION
**Pull request by onlyabdiel**
The sidebar was comleted and the webpack config is completed

### CODE REVIEW

File: DesignSystem/src/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+@import url('https://fonts.googleapis.com/css2?family=Jost:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;700&display=swap');
+
+// Colors
+
+$primary-black: #020202;
+$primary-white: #F6FAFF;
+$secondary-blue: #3255AD;
+$secondary-gray: #CCB89D;
```
Comment (WS-Jedp) [inline DesignSystem/src/styles/variables.scss]:
```suggestion:
$color-primary-black: #020202;
$primary-black: #020202;
$primary-white: #F6FAFF;
$secondary-blue: #3255AD;
$secondary-gray: #CCB89D;
```

File: DesignSystem/src/styles/variables.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+// Fonts
+
+$primary-font: 'Jost', sans-serif;
+$secondary-font: 'Josefin Sans', sans-serif;
```
Comment (WS-Jedp) [inline DesignSystem/src/styles/variables.scss]:
```suggestion:

--> $font-primary: 'Jost', sans-serif;

$primary-font: 'Jost', sans-serif;
$secondary-font: 'Josefin Sans', sans-serif;
```

File: DesignSystem/src/styles/variables.scss (hunk 1 above)
Comment (onlyabdiel) [reply DesignSystem/src/styles/variables.scss]: issues solved

File: DesignSystem/src/styles/variables.scss (hunk 2 above)
Comment (onlyabdiel) [reply DesignSystem/src/styles/variables.scss]: issues solved

**Pull request merged**